### PR TITLE
Removes set-pose and pose text

### LINKED ISSUE #13871: Set-pose verb is flavor text but stronger
Use the set-pose verb
Enter LITERALLY ANYTHING unlogged, have said text show up on examine through any disguise or identity dealio
image

(said player was actually Jackie Chan disguised as someone else, but the text still showed)

### PR DESCRIPTION
**Pull request by gbasood**
Resolves #13871 
We're still not an RP server and even if we were this is so poorly implemented.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -471,10 +471,6 @@
 			<span class = 'deptradio'>Medical records:</span> <a href='?src=\ref[src];medrecord=`'>\[View\]</a> <a href='?src=\ref[src];medrecordadd=`'>\[Add comment\]</a>\n"}
 
 	msg += "*---------*</span>"
-	if (pose)
-		if( findtext(pose,".",length(pose)) == 0 && findtext(pose,"!",length(pose)) == 0 && findtext(pose,"?",length(pose)) == 0 )
-			pose = addtext(pose,".") //Makes sure all emotes end with a period.
```
Comment (gbasood) [inline code/modules/mob/living/carbon/human/examine.dm:476]: hoo boy let's just do this all on EVERY EXAMINE

**Review by PJB3005 — APPROVED** [review]

**Comment by kilozombie** [comment]
How am I supposed to notify my friends that Kayleigh is a vampire silently now??

**Comment by Kuranon** [comment]
How hard is it to add admin logging?


>We're still not an RP server
Light RP is still RP!

**Comment by ThePlapatin** [comment]
removing things isn't the best way to do this, if you add admin logging it lets admins know when it is being used (maybe once a year)

just like with ghost light flicking, if it is logged people can be caught if they use it to metagame I think

**Comment by Probe1** [comment]
muh metacommunication roleplay

**Comment by gbasood** [comment]
@Kuranon not even about logging
Think of a reason this is actually good and then get someone to readd it for that if you think it's worthwhile

**Comment by Kuranon** [comment]
Then what is it about?

What did @Probe1  mean by that?

**Comment by gbasood** [comment]
He meant that and I mispoke somewhat
Logging is an important part but this is literally only useful for metaknowledge
Like flavortext but you can change it EASILY

**Comment by kol1th** [comment]
@Kuranon This isn't even a good feature on RP servers, if you're wondering.

**Comment by ThePlapatin** [comment]
flavortext back when then?

if this is just shitty flavortext, can we have good flavortext back?

also
>merging in an hour of opening the pr

**Comment by Probe1** [comment]
>what did he mean by this

Emojiocracy doesn't outweigh exploits you silly goose.

**Comment by gbasood** [comment]
See above statement about being worthwhile and getting someone to readd it

**Comment by kol1th** [comment]
This nor flavor text adds nothing to the server. This isn't a roleplay server.

**Comment by TheFriggenSawbones** [comment]
i think it's time we ban Plapatin from the github

**Comment by ThePlapatin** [comment]
"exploits"
fucking bay manages to tell people the round type and their admins handle any metagaming, are ours just plain lazy?

**Comment by kol1th** [comment]
Yes, because powergaming and such isn't bannable here. Do you play on Bay often? Or any roleplay server?

**Comment by Probe1** [comment]
@ThePlapatin idk if you've gotten the memo yet but I pretty much breeze over anything you ever write.  Associate your actual ckey if you expect to be listened to on anything.

**Comment by ThePlapatin** [comment]
@kol1th I've played on bay a few times, it's not that far off from /vg/. Powergamers are a very small population of our server, why do the many suffer because of the few?

@Probe1 You want people to interact with the github and then don't listen to them? Why would you use your ckey on github, you'll become a meme like yclat if you disagree with the coders too many times. Does goof even play on our server and he is always on this github.

**Comment by gbasood** [comment]
@ThePlapatin believe it or not they have a different playerbase and admin philosophy than we do

**Comment by ThePlapatin** [comment]
@gbasood Eh that's true, I don't think we have many metagamers or powergamers though, it's exaggerated

**Comment by JustSumBody** [comment]
>why would you use your ckey on github

Why would you use a permabanned autist's ckey instead

**Comment by Probe1** [comment]
I'm not debating anything with you Plapatin I'm telling you that I never listen to anything you say because you hide behind a permabanned players name.  It's up to you to behave like you're not on 4chan.

**Comment by gbasood** [comment]
http://ss13.moe/index.php/bans ctrl+f meta

**Comment by ThePlapatin** [comment]
@JustSumBody You'd rather I made my github name into something like "648768767836"?, if it makes it so people don't just ignore what I write out of habit I'd be happy to
@gbasood a lot of bans say meta multiple times which makes the count seem higher than normal. and most meta bans are from obvious stuff like skype calls/working really closely together or using ghosting knowledge they shouldn't have. I think what I meant to say is that if somebody was going to metagame, they have much much better options. Everyone has steam and the admins can't see your computer, it's much safer and way more reliable

**Comment by gbasood** [comment]
sure as long as they know the person they're trying to communicate with
"Help I'm silence stung and X is a ling" is a possible outcome of this verb though

also no really your use of another banned player's name is dumb

**Comment by ThePlapatin** [comment]
I guess so, it just seems like a very limited way to metagame. It's only within a screen of radius, requires you to examine them and can't be done verbally.

Is it like flavortext where you set it once and it lasts the whole round? If so it could be bad if say a player always sets it to one thing, then somebody attempts to impersonate them and people know that they're a fake as they don't have the post text?
if it is like that I can understand why it's being removed, I thought it was more of a limited time thing as you wouldn't be posed in a way for a whole shift.

I'll trust you on this sood, you're alright

**Comment by kol1th** [comment]
>commenting in a merged pr


kys

**Comment by gbasood** [comment]
It lasts per mob so yeah your corpse will still have it even when husked probably
Lings might actually not have it persist between morphs? But still

**Comment by PJB3005** [comment]
This is a stupid discussion. Kill it.

**Pull request merged**